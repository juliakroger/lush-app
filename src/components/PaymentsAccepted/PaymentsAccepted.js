import React from 'react'
import styles from './PaymentsAccepted.module.scss'
import { Icon } from '..'

const PaymentsAccepted = () => (
  <div className={styles.row}>
    <span className={styles.text}>We accept:</span>
    <div className={styles.icon}>
      <Icon type='visa' className={styles.card} />
      <Icon type='amex' className={styles.card} />
      <Icon type='mastercard' className={styles.card} />
      <Icon type='paypal' className={styles.card} />
    </div>
  </div>
)

export default PaymentsAccepted
