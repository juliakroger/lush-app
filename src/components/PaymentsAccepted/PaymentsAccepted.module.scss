@import '../../scss/variables.scss';
@import '../../scss/responsive-utils';

.row {
  display: flex;
  flex-direction: row;
  padding-top: 20px;

  .text {
    margin-top: 8px;
    padding-right: 6px;
  }

  .card {
    padding-right: 30px !important;
  }

  .icon {
    span {
      padding-right: 6px !important;
    }
  }
}
