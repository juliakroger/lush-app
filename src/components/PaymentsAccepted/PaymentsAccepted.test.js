import React from 'react'
import { shallow } from 'enzyme'
import PaymentsAccepted from './PaymentsAccepted'

describe('Component - Payments Accepted', () => {
  it('should render without crashing', () => {
    const component = shallow(<PaymentsAccepted />)
    expect(component).toMatchSnapshot()
  })
})
