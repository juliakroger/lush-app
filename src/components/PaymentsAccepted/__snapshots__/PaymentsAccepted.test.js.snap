// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component - Payments Accepted should render without crashing 1`] = `
<div
  className="row"
>
  <span
    className="text"
  >
    We accept:
  </span>
  <div
    className="icon"
  >
    <Icon
      className="card"
      type="visa"
    />
    <Icon
      className="card"
      type="amex"
    />
    <Icon
      className="card"
      type="mastercard"
    />
    <Icon
      className="card"
      type="paypal"
    />
  </div>
</div>
`;
